index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse!</title>
+</head>
+
+<body>
+    <h1>Hello Microverse!</h1>
```

Comment:
According to the project requirement, you are to add a class to your `header` and then style the class in your css file, please add a class in the `h1` tag and then go ahead with the styling in the css file 😄 

![image](https://github.com/benjaMatteo96/hello-microverse-n1/assets/20537712/da2106d3-df14-4c1a-83fd-1fc23084f02a)